<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>409</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#widgetTitle{
border-width: 0px  0px 0px 280px;
border-image: url(:/title.png) 0px 0px 0px 280px;
}
QWidget#widgetPreview{
border: none;
background-color: rgb(0, 0, 0);
}
QWidget#widgetBottom
{
background-color: rgb(64, 64, 64,160);
}

QWidget#stackedWidgetAddContents{
border: none;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0.213542 rgba(0, 0, 0, 255), stop:1 rgba(77, 77, 77, 224));
}

QWidget#widgetContents{
border: none;
background-color: rgb(64, 64, 64,160);
}
QWidget#listViewContents{
border: none;
background-color: rgb(64, 64, 64,0);
}
QPushButton{
outline: none;
}
QPushButton#pushButtonRecStart{
border-width: 0px  0px 0px 0px;
border-image: url(:/butStart_normal.png);
}

QPushButton#pushButtonRecStart:hover{
border-width: 0px  0px 0px 0px;
border-image: url(:/butStart_enter.png);
}
QPushButton#pushButtonRecStart:disabled{
border-width: 0px  0px 0px 0px;
border-image: url(:/butStart_disable.png);
}

QPushButton#pushButtonRecStop{
border-width: 0px  0px 0px 0px;
border-image: url(:/butStop_normal.png);
}
QPushButton#pushButtonRecStop:hover{
border-width: 0px  0px 0px 0px;
border-image: url(:/butStop_enter.png);
}
QPushButton#pushButtonRecStop:disabled{
border-width: 0px  0px 0px 0px;
border-image: url(:/butStop_disable.png);
}

QPushButton#pushButtonRecPause{
border-width: 0px  0px 0px 0px;
border-image: url(:/butPause_normal.png);
}
QPushButton#pushButtonRecPause:hover{
border-width: 0px  0px 0px 0px;
border-image: url(:/butPause_enter.png);
}
QPushButton#pushButtonRecPause:disabled{
border-width: 0px  0px 0px 0px;
border-image: url(:/butPause_disable.png);
}

QPushButton#pushButtonRecPause:checked{
border-width: 0px  0px 0px 0px;
border-image: url(:/butPause_checked_normal.png);
}
QPushButton#pushButtonRecPause:checked:hover{
border-width: 0px  0px 0px 0px;
border-image: url(:/butPause_checked_enter.png);
}
QPushButton#pushButtonRecPause:checked:disabled{
border-width: 0px  0px 0px 0px;
border-image: url(:/butPause_checked_disable.png);
}

QLabel#labelRecordTime
{
color: rgb(200, 200, 200);
font-size:14px;

}

QPushButton#pushButtonScreenSelect
{
border: none;
color: rgb(200, 200, 200);
border-left: 1px  solid  rgb(64, 64, 64);
}
QPushButton#pushButtonScreenSelect:hover
{
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
}
QPushButton#pushButtonScreenSelect:checked
{
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
}
QPushButton#pushButtonCameraSelect
{
border: none;
color: rgb(200, 200, 200);
border-left: 1px  solid  rgb(64, 64, 64);
}
QPushButton#pushButtonCameraSelect:hover
{
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
}
QPushButton#pushButtonCameraSelect:checked
{
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
}
QPushButton#pushButtonMediaSelect
{
border: none;
color: rgb(200, 200, 200);
border-left: 1px  solid  rgb(64, 64, 64);
border-right: 1px  solid  rgb(64, 64, 64);
}
QPushButton#pushButtonMediaSelect:hover
{
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
}
QPushButton#pushButtonMediaSelect:checked
{
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
}

QPushButton#pushButtonClose
{
border: 1px solid rgb(160, 160, 160);
border-radius: 5px;
color: rgb(160, 160, 160);
}
QPushButton#pushButtonClose:hover
{
background-color: rgb(240, 240, 240);
border: 1px solid rgb(240, 240, 240);
color: rgb(160, 160, 160);
}

QPushButton#pushButtonMinimize
{
border: 1px solid rgb(160, 160, 160);
border-radius: 5px;
color: rgb(160, 160, 160);
}
QPushButton#pushButtonMinimize:hover
{
background-color: rgb(240, 240, 240);
border: 1px solid rgb(240, 240, 240);
color: rgb(160, 160, 160);
}

QPushButton#pushButtonMenu
{
border: 1px solid rgb(160, 160, 160);
border-radius: 5px;
color: rgb(160, 160, 160);
}

QPushButton#pushButtonMenu:hover
{
background-color: rgb(240, 240, 240);
border: 1px solid rgb(240, 240, 240);
color: rgb(160, 160, 160);
}

QLabel#labelRenderFps
{
color: rgb(160, 160, 160);
}
QLabel#labelVideoInfo
{
color: rgb(160, 160, 160);
}

</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widgetTitle" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>80</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widgetLogo" native="true">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <widget class="QLabel" name="labelLogo">
          <property name="geometry">
           <rect>
            <x>4</x>
            <y>8</y>
            <width>128</width>
            <height>64</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>64</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>128</width>
            <height>64</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="UiResource/recorder.qrc">:/logo.png</pixmap>
          </property>
         </widget>
         <widget class="QLabel" name="labelVersion">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>61</y>
            <width>151</width>
            <height>22</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(20, 60, 90);
font-size:8px;
</string>
          </property>
          <property name="text">
           <string>V1.0.0 beta [2020-09-20]</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidgetRecControl">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>80</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>80</height>
          </size>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>12</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButtonRecStart">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>70</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>70</height>
              </size>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="toolTip">
              <string>点击按钮开始录像</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButtonRecStop">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="toolTip">
              <string>停止录像</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>4</number>
             </property>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonRecPause">
               <property name="minimumSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="focusPolicy">
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="toolTip">
                <string>暂停、继续录像</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelRecordTime">
               <property name="text">
                <string>00:00:00</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QWidget" name="widgetTitleInfo" native="true">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>24</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>24</height>
              </size>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="labelRenderFps">
                <property name="text">
                 <string>-</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="labelVideoInfo">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>TextLabel</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonMenu">
             <property name="maximumSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="toolTip">
              <string>打开设置界面</string>
             </property>
             <property name="text">
              <string>=</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonMinimize">
             <property name="minimumSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="toolTip">
              <string>窗口最小化</string>
             </property>
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonClose">
             <property name="minimumSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="toolTip">
              <string>关闭程序</string>
             </property>
             <property name="text">
              <string>X</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QWidget" name="widget" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="pushButtonScreenSelect">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>90</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>添加屏幕、屏幕区域、窗口</string>
                </property>
                <property name="text">
                 <string>屏幕</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonCameraSelect">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>90</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>添加摄像头</string>
                </property>
                <property name="text">
                 <string>摄像头</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButtonMediaSelect">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>90</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>添加图像文件</string>
                </property>
                <property name="text">
                 <string>图像</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="FormAudioRec" name="widget_AudioRec" native="true">
             <property name="minimumSize">
              <size>
               <width>140</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="GlWidgetPreview" name="widgetPreview" native="true">
      <property name="toolTip">
       <string>鼠标左键选择图层，鼠标右键放弃选择</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignTop">
        <widget class="StackedWidgetAddLayer" name="stackedWidgetAddContents">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>140</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="toolTip">
          <string>添加画面内容为图层</string>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="page_3"/>
         <widget class="QWidget" name="page_4"/>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item alignment="Qt::AlignBottom">
     <widget class="QWidget" name="widgetBottom" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>4</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>4</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GlWidgetPreview</class>
   <extends>QWidget</extends>
   <header>GlWidgetPreview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StackedWidgetAddLayer</class>
   <extends>QStackedWidget</extends>
   <header>StackedWidgetAddLayer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormAudioRec</class>
   <extends>QWidget</extends>
   <header>FormAudioRec.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButtonScreenSelect</tabstop>
  <tabstop>pushButtonCameraSelect</tabstop>
  <tabstop>pushButtonMediaSelect</tabstop>
  <tabstop>pushButtonRecStop</tabstop>
  <tabstop>pushButtonRecPause</tabstop>
 </tabstops>
 <resources>
  <include location="UiResource/recorder.qrc"/>
 </resources>
 <connections/>
</ui>
